.travis.yml

```diff
@@ -1,3 +1,4 @@
+dist: trusty
```

Comment:
What is this for?

Reply:
I am not sure. When I did not have this there, the [travis process](https://travis-ci.com/LiveRamp/captain/builds/111505854) errored on setting up JDK8. I could try taking it out again if you think it's not necessary.

Reply:
Oh i see. Eh, was mostly just wondering why this exists. I'll trust you on this 😆